optimade/validator/__init__.py

```diff
@@ -39,7 +39,7 @@ def validate():
         ),
     )
     parser.add_argument(
-        "--verbosity", "-v", type=int, default=1, help="The verbosity of the output"
+        "--verbosity", "-v", type=int, default=0, help="The verbosity of the output"
```

Comment:
I can see we may need to think of setting these through environment variables as well so I can get me some permanent verbosity level of 1 😅

Reply:
I can get it to respond to `OPTIMADE_VERBOSITY` (which can be reused), just for you!